test/action_controller/json_api/linked_test.rb

```diff
@@ -43,29 +43,29 @@ def render_resource_without_include
 
           def render_resource_with_include
             setup_post
-            render json: @post, include: 'author', adapter: :json_api
+            render json: @post, include: [:author], adapter: :json_api
```

Comment:
These are the only tests involving `action_controller` + `include` would be nice to have some of them or new ones covering the string behavior in to make sure we con't break it in the future.